### PR TITLE
Feature/IL-300/scrollup

### PR DESCRIPTION
**Pull request by iakivpekarskyi**
- added button to scroll up to the top of the screen
 - removed scale on buttons
 - minor fixes with margins

### CODE REVIEW

**Review by DmitriyUshkvarok — CHANGES_REQUESTED** [review]
the button after adding an item to cart should contain the text that the item is in cart and be locked
addcart

File: src/components/Favourites/FavouritesFull/FavouritesFull.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ export default function FavouritesFull() {
       </div>
       <div className="flex w-full justify-center">
         <Link href={'/'}>
-          <Button className="my-6 h-14  text-lg font-medium sm:w-[211px]">
+          <Button className="my-6 h-14 text-lg font-medium sm:w-[211px]">
```
[the remark this answers is not in the document]
Comment (iakivpekarskyi) [reply src/components/Favourites/FavouritesFull/FavouritesFull.tsx:31]: fixed

File: src/components/UI/Buttons/ScrollUpBtn/ScrollUpBtn.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    window.scrollTo({
+      top: 0,
+      behavior: 'smooth'
+    })
+  }
+
+  return (
+    <>
+      {showButton && (<button className="m-2 fixed bottom-2 right-2 bg-secondary bg-opacity-90 p-1 rounded-full w-14 h-14 transition-all duration-900 ease-in-out transform hover:-translate-y-1 hover:scale-110 border-gray-300 border-2" onClick={scrollToTop}><Image src={arrowup} alt="up_arrow" className={'inline-block'} /></button>)}
+    </>
```
[the remark this answers is not in the document]
Comment (iakivpekarskyi) [reply src/components/UI/Buttons/ScrollUpBtn/ScrollUpBtn.tsx:29]: fixed

**Comment by iakivpekarskyi** [comment]
> the button after adding an item to cart should contain the text that the item is in cart and be locked addcart

when item is added this button splits into two parts indicating that item was added

**Review by DmitriyUshkvarok — APPROVED** [review]

**Pull request merged**